docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "2.4"
+services:
+    elastic:
+        image: docker.elastic.co/elasticsearch/elasticsearch:7.5.0
+        ports:
+            - "9200:9200"
+        environment:
+            - discovery.type=single-node
+        tmpfs:
+            - /usr/share/elasticsearch/data
+
+    redis:
+        image: redis:3
+        ports:
+            - "6379:6379"
+
+    mongo:
+        image: mongo:3.4
+        ports:
+            - "27017:27017"
+        tmpfs:
+            - /data/db
+
+    rabbitmq:
```

Comment:
Why do we need a rabbitmq ?

Reply:
we don't, I just wanted to run tests in environment close to production

Reply:
but it's probably not used anyhow due to eager celery, so will remove it